stock_landed_cost_company_percentage/models/stock_picking.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    def prepare_landed_cost_lines(self, company, amount):
+        cost_lines = []
+        for line in company.landed_cost_company_line:
+            values = {
+                'product_id': line.product_id.id,
+                'split_method': line.product_id.split_method,
+                'account_id': line.product_id.property_account_expense_id.id or
+                              line.product_id.categ_id.property_account_expense_categ_id.id, # noqa
```

Comment:
Instead of disabling the lint, you should use an intermediate variable.

```python
for line in company.landed_cost_company_line:
    product = line.product_id
    account = (
        product.property_account_expense_id or
        product.categ_id.property_account_expense_categ_id
    )
    values = {
        'product_id': product.id,
        ...
```

Reply:
Corresponded. Thanks.